x-pack/plugins/security_solution/cypress/screens/timeline.ts

```diff
@@ -118,6 +118,8 @@ export const TIMELINE_DATA_PROVIDERS = '[data-test-subj="dataProviders"]';
 
 export const TIMELINE_DATA_PROVIDERS_ACTION_MENU = '[data-test-subj="providerActions"]';
 
+export const TIMELINE_ADD_FIELD_BUTTON = '[data-test-subj="addField"]';
```

Comment:
is this selector still required?

Reply:
ah, yeah, I think it makes the drop area clearer. I'll use it as the drop target.